7.1.Цикл_for/7.1.3.py

```diff
@@ -1,5 +1,5 @@
-# Повторяй за мной 1
-
-text = input()
-for i in range(int(input())):
-    print(text)
\ No newline at end of file
+# put your python code here
+text = input()
+for i in range(int(input())):
+    print(text)
```

Comment:
Так конечно можно делать, но лучше кол-во повторений отдельно объявить:
```suggestion
text = input()
repeat = int(input())
for i in range(repeat):
    print(text)
```